### DISCUSSION TITLE
Data importer: how to import more than one media into a many-to-many relationship field

### DISCUSSION THREAD

**Discussion by trnqeu** [pimcore/pimcore#16915] [Q&A / Support]
Hi.
I'm having an issue with the Pimcore importer. 

In my Excel file I have a column where I have a list of Pimcore media files.
I want to relate those media to a Pimcore object, specifically using a many-to-many relationship field.

I find out a way to load a single file but when I try to load more than an asset something goes wrong.
This is a screenshot of my mapping:

![image](https://github.com/pimcore/pimcore/assets/48558947/1696faa2-5cd8-4b05-b8b4-200c43a4354f)

Any idea?

**Comment by LSBelperTech** [comment]
Hi, did you ever figure out a solution for this? Currently running into the same problem myself.

**Reply by trnqeu** [reply]
Unfortunately not. I'm looking into this plugin: https://pimcore.blackbit.com/en/solutions/data-director
